src/python/pants/core/goals/multi_tool_goal_helper.py

```diff
@@ -74,12 +74,12 @@ def __new__(cls, uppercase: str, lowercase: str):
         )
 
 
-def determine_specified_tool_names(
+def determine_specified_tool_slugs(
     goal_name: str,
     only_option: Iterable[str],
     all_requests: Iterable[type],
 ) -> set[str]:
-    all_valid_names = {request.tool_name for request in all_requests}  # type: ignore[attr-defined]
+    all_valid_names = {request.tool_slug for request in all_requests}  # type: ignore[attr-defined]
```

Comment:
I don't change the user-facing string, because they really don't need to know the difference between "name" and "slugs". As far as they care, it's a name.

Alternatively we could call the property `tool_programmatic_name`.

Reply:
I am 👍 to Eric's suggestion of `tool_id` as an alternate to `tool_slug`

Reply:
To be very clear, we'd keep the user-facing string as using `name` though, right?